packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -177,13 +177,23 @@ async function reconciliationReportForGranuleFiles(granuleInDb, granuleInCmr, bu
       // filename in both cumulus and CMR
       if (granuleFiles[urlFileName] && bucketsConfig.key(granuleFiles[urlFileName].bucket)) {
         // not all files should be in CMR
-        const accessUrl = constructOnlineAccessUrl({
+        const distributionAccessUrl = constructOnlineAccessUrl({
           file: granuleFiles[urlFileName],
           distEndpoint: process.env.DISTRIBUTION_ENDPOINT,
-          buckets: bucketsConfig
+          buckets: bucketsConfig,
+          cmrGranuleUrlType: 'distribution'
         });
 
-        if (accessUrl && relatedUrl.URL === accessUrl.URL) {
+        const s3AccessUrl = constructOnlineAccessUrl({
+          file: granuleFiles[urlFileName],
+          distEndpoint: process.env.DISTRIBUTION_ENDPOINT,
+          buckets: bucketsConfig,
+          cmrGranuleUrlType: 's3'
+        });
+
+        if (distributionAccessUrl && relatedUrl.URL === distributionAccessUrl.URL) {
```

Comment:
Just to make sure I’m reading this correctly. If we have granules in CMR that were uploaded with some kind of URL (distribution or s3) and then the Cumulus stack is updated to have a `cmrGranuleUrlType` of `none`, then we’d expect the following reconciliation report to have no overlap.

Similarly, if granules are uploaded to CMR with a `distribution` URL and then the Cumulus stack is updated to use a `cmrGranuleUrlType` of `s3`, then we’d expect those granules with a `distribution` URL in CMR to not match, correct?

Reply:
So it's your workflow that gets updated, not the stack, which is why we don't look at that here. 

So whatever you use, if you have some kind of link it should match, if you have none, it will show up in reconciliation.

I confuse myself talking about it so if you want to chat about it on Monday, we can.

Reply:
Just spoke to Lauren about this. Since the workflow is what's being updated, changes to `cmrGranuleUrlType` are really only realized in the individual granule's metadata that pass through that `moveGranules` step.

That is, this shouldn't have the effect I suggested on reconciliation.